src/palace/manager/feed/serializer/opds.py

```diff
@@ -55,18 +62,16 @@
 }
 
 
-def is_sort_link(link: Link) -> bool:
-    """A until method that determines if the specified link is a sort link"""
+def is_sort_facet(link: Link) -> bool:
+    """A until method that determines if the specified link is part of a sort facet"""
     return (
         hasattr(link, "facetGroup")
         and link.facetGroup
         == FacetConstants.GROUP_DISPLAY_TITLES[FacetConstants.ORDER_FACET_GROUP_NAME]
     )
 
 
-class OPDS1Serializer(SerializerInterface[etree._Element], OPDSFeed):
-    """An OPDS 1.2 Atom feed serializer"""
-
+class BaseOPDS1Serializer(SerializerInterface[etree._Element], OPDSFeed):
```

Comment:
You also need to inherent from ABC in order to make this an abstract base class.

See: 
https://docs.python.org/3/library/abc.html#abc.ABC